typings/git.d.ts

```diff
@@ -0,0 +1,329 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Do we really need to copy this typings from vscode?
Is there no way to import them?